exercises/exercise4.js

```diff
@@ -0,0 +1,32 @@
+//  function that checks the level of nesting of an array
+function createSingleDimensionArray() {
+  let arr1 = [1, 2, 3, [4, 5, [6, 7]]];
+  console.log(arr1.length);
+  return arr1.flat(Infinity);
+}
+
+// function to check if a number is prime or not
+function isPrime(n) {
```

Comment:
- prime numbers are positive numbers only.
- this function returns "undefined" when 2 or 3 is passed as argument
- The naming of the parameter is not meaningful